include/xplat/IwMalloc.h

```diff
@@ -35,7 +35,13 @@ inline char* IwStrDup(const char* str) {
     char* buffer = (char*)IwMalloc(len + 1);
 
     if (buffer != NULL) {
+// strcpy_s is a MSVC extension,
+// strncpy is not usable in our current WinObjC build
+#ifdef _WIN32
```

Comment:
Not here - elsewhere yes.

---
In reply to: [72322094](https://github.com/Microsoft/WinObjC/pull/634#discussion_r72322094) [](ancestors = 72322094)